models/base/scratch/snowplow_mobile_base_sessions_this_run.sql

```diff
@@ -1,23 +1,27 @@
-{{ 
+{{
   config(
     sort='start_tstamp',
     dist='session_id',
+    partition_by = snowplow_utils.get_partition_by(bigquery_partition_by={
+       "field": "start_tstamp",
+       "data_type": "timestamp"
+     }, databricks_partition_by='start_tstamp'),
```

Comment:
Same here tbh, anywhere where we partition by `start_tstamp`

Reply:
removed here as it is just a this run table (we don't have partitioning for the events this run either